core-bundle/src/Filesystem/FilesystemItemIterator.php

```diff
@@ -17,6 +17,17 @@
  */
 class FilesystemItemIterator implements \IteratorAggregate
 {
+    public const SORT_BY_PATH_ASC = 'name_asc';
+    public const SORT_BY_PATH_DESC = 'name_desc';
+    public const SORT_BY_LAST_MODIFIED_ASC = 'date_asc';
+    public const SORT_BY_LAST_MODIFIED_DESC = 'date_desc';
+
+    public static array $supportedSortingModes = [
```

Comment:
We could use PHP 8.1 backed enums here, maybe? Then the modes are all in sync and we could use `EnumClass:: cases()` 🥳

Reply:
Good point! We should definitively do that.

Reply:
see bdf3056b22ddbed0d073460602a271e757170a92